README.md

```diff
@@ -962,14 +962,18 @@ const winston = require('winston');
 //
 // Configure the logger for `category1`
 //
+const { format } = winston;
+const { combine, label, json } = format;
+
 winston.loggers.add('category1', {
-  console: {
-    level: 'silly',
-    label: 'category one'
-  },
-  file: {
-    filename: '/path/to/some/file'
-  }
```

Comment:
When I tried using the above setup with:

```js
const category1 = winston.loggers.get('category1');

category1.info('logging from your IoC container-based logger');
```

I received the following error:

```js
[winston] Attempt to write logs with no transports {"message":"logging from your IoC container-based logger","level":"info"}
```

Am I doing something wrong, or is this documentation incorrect? I've updated the docs to include code that works as of Winston 3.1.0.

Reply:
done. Let me know if you want me to squash the commits